src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -551,6 +551,68 @@ ObjectDataHandler::read_ret ObjectDataHandler::read(
     });
 }
 
+ObjectDataHandler::fiemap_ret ObjectDataHandler::fiemap(
+  context_t ctx,
+  objaddr_t obj_offset,
+  extent_len_t len)
+{
+  return seastar::do_with(
+    std::map<uint64_t, uint64_t>(),
+    [ctx, obj_offset, len](auto &ret) {
+    return with_object_data(
+      ctx,
+      [ctx, obj_offset, len, &ret](const auto &object_data) {
+      LOG_PREFIX(ObjectDataHandler::fiemap);
+      DEBUGT("fiemap {}~{}",
+              ctx.t,
+              object_data.get_reserved_data_base(),
+              object_data.get_reserved_data_len());
```

Comment:
Sorry, another change here since I found this confusing when I was debugging.

```
       LOG_PREFIX(ObjectDataHandler::fiemap);
       DEBUGT(
        "{}~{}, reservation {}~{}",
        ctx.t,
        obj_offset,
        len,
        object_data.get_reserved_data_base(),
        object_data.get_reserved_data_len());
```